api.visual/apichanges.xml

```diff
@@ -29,7 +29,20 @@
     </apidefs>
 
     <changes>
-
+        <change>
+            <api name="general"/>
+            <summary>States for non-registered objects</summary>
+            <version major="2" minor="49"/>
+            <date day="27" month="9" year="2017"/>
+            <author login="sdedic"/>
+            <compatibility addition="yes"/>
+            <description>
+                When the scene is incrementally rendered, it may be possible that a model object becomes selected before some widgets are registered with the scene.
+                Hover, select, focus flags can be attached even to unregistered objects.
+            </description>
+            <class name="ObjectScene" package="org.netbeans.api.visual.model" link="yes"/>
```

Comment:
Here we should reference this [PR-308](https://github.com/apache/incubator-netbeans/pull/308/). We may need to enance the `issue` tag functionality to do so.

Reply:
Good point. I have created https://issues.apache.org/jira/browse/NETBEANS-192 to track the idea; for now, I'll reference the PR in plaintext.

Reply:
Hi, would'nt be easier to create a JIRA ticket for this PR and then only reference to Apache NetBeans JIRA instead of github link ?
Allowing issue tag to keep semantic link to issue instead "list of commit"